src/dos/dos_programs.cpp

```diff
@@ -327,31 +327,8 @@ class MOUNT : public Program {
 				int num = -1;
 				cmd->FindInt("-usecd",num,true);
 				int error = 0;
-				if (cmd->FindExist("-ioctl_dio", false)) {
-					MSCDEX_SetCDInterface(CDROM_USE_IOCTL_DIO, num);
-				} else if (cmd->FindExist("-ioctl_dx",false)) {
-					MSCDEX_SetCDInterface(CDROM_USE_IOCTL_DX, num);
-#if defined (WIN32)
-				} else if (cmd->FindExist("-ioctl_mci",false)) {
-					MSCDEX_SetCDInterface(CDROM_USE_IOCTL_MCI, num);
-#endif
-				} else if (cmd->FindExist("-noioctl",false)) {
+				if (cmd->FindExist("-noioctl",false)) {
 					MSCDEX_SetCDInterface(CDROM_USE_SDL, num);
-				} else {
-#if defined (WIN32)
-					// Check OS
-					OSVERSIONINFO osi;
-					osi.dwOSVersionInfoSize = sizeof(osi);
-					GetVersionEx(&osi);
-					if ((osi.dwPlatformId==VER_PLATFORM_WIN32_NT) && (osi.dwMajorVersion>5)) {
-						// Vista/above
-						MSCDEX_SetCDInterface(CDROM_USE_IOCTL_DX, num);
-					} else {
-						MSCDEX_SetCDInterface(CDROM_USE_IOCTL_DIO, num);
-					}
-#else
-					MSCDEX_SetCDInterface(CDROM_USE_IOCTL_DIO, num);
-#endif
 				}
```

Comment:
Previously by default, `mount d f:\ -t cdrom` would be an `-ioctl` mount via IOCTL_DX or DIO, which would give them an MSCDEX interface.

With this gone now, SDL is the only remaining option to get an MSCDEX interface; so suggest we make `MSCDEX_SetCDInterface(CDROM_USE_SDL, num);` the default here if nothing is specified. 
If `-ioctl` is specified, then maybe exit with an error?